package.json

```diff
@@ -32,7 +32,7 @@
     "webdriverio": "~6 || ~7 || ^8.6.6"
```

Comment:
https://github.com/percy/percy-webdriverio/pull/1113/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L29 also need to be updated